pkg/convert/supportbundle.go

```diff
@@ -70,7 +70,7 @@ func FromAnalyzerResult(input []*analyze.AnalyzeResult) []*Result {
 	result := make([]*Result, 0)
 	for _, i := range input {
 		// Continue on nil result to prevent panic
-		if i == nil {
+		if i == nil || i.Title == "" {
```

Comment:
I assume `i.Title == ""` is meant to catch the empty Result case. I think an analyser returning a `nil` result is a better approach here to indicate that there is no analysis result.

Reply:
I have removed `i.Title == ""` and return a nil analysis  result